server/unittools.cpp

```diff
@@ -2836,6 +2836,11 @@ void send_unit_info(struct conn_list *dest, struct unit *punit)
       if (pdata != nullptr) {
         BV_SET(pdata->can_see_unit, player_index(pplayer));
       }
+    } else if (pplayer->team == powner->team) {
```

Comment:
To facilitate maintenance I would prefer this (from `player.h`):
```suggestion
    } else if (players_on_same_team(pplayer, powner)) {
```
Could we merge this with the previous `if`? After all, a player is always on its own team...